overlord/devicestate/devicemgr.go

```diff
@@ -78,6 +78,8 @@ func Manager(s *state.State, hookManager *hookstate.HookManager, runner *state.T
 	runner.AddHandler("generate-device-key", m.doGenerateDeviceKey, nil)
 	runner.AddHandler("request-serial", m.doRequestSerial, nil)
 	runner.AddHandler("mark-seeded", m.doMarkSeeded, nil)
+	// FIXME: do we need an undo handler here?
```

Comment:
I think we need strive to make this always the last task in relevant changes. We cannot undo this operation really if it's change a revision change (there is no revert for assertions). That means tough we probably need a comment here and were we add about this fact that it *must* be last.